mission/mission.proto

```diff
@@ -6,25 +6,63 @@ option java_package = "io.dronecore.mission";
 option java_outer_classname = "MissionProto";
 
 service MissionService {
-    rpc SendMission(SendMissionRequest) returns(MissionResult) {}
-    rpc StartMission(StartMissionRequest) returns(MissionResult) {}
+    rpc UploadMission(UploadMissionRequest) returns(UploadMissionResponse) {}
+    rpc DownloadMission(DownloadMissionRequest) returns(DownloadMissionResponse) {}
+    rpc StartMission(StartMissionRequest) returns(StartMissionResponse) {}
+    rpc PauseMission(PauseMissionRequest) returns(PauseMissionResponse) {}
+    rpc SetCurrentMissionItemId(SetCurrentMissionItemIdRequest) returns(SetCurrentMissionItemIdResponse) {}
+    rpc GetCurrentMissionItemId(GetCurrentMissionItemIdRequest) returns(GetCurrentMissionItemIdResponse) {}
+    rpc GetMissionSize(GetMissionSizeRequest) returns(GetMissionSizeResponse) {}
+    rpc SubscribeMissionProgress(SubscribeMissionProgressRequest) returns(stream MissionProgressResponse) {}
```

Comment:
We need one for [mission_finished()](https://github.com/dronecore/DroneCore/blob/904ba588f554562672e79ea9dcc9449096219d47/plugins/mission/mission.h#L125).
Probably,
```
message IsMissionDoneRequest {}
message IsMissionDoneResponse {
     bool is_done = 1;
 }

rpc IsMissionDone(IsMissionDoneRequest) returns(IsMissionDoneResponse) {}
```

Reply:
`IsMissionFinished`, for consistency?

Reply:
Yeah, fine.